web/src/test/java/org/springframework/security/web/authentication/www/BasicAuthenticationConverterTests.java

```diff
@@ -111,4 +112,12 @@ public void convertWhenEmptyPassword() {
 		assertThat(authentication.getName()).isEqualTo("rod");
 		assertThat(authentication.getCredentials()).isEqualTo("");
 	}
+
+	@Test
```

Comment:
Please add the expected exception to the `@Test` annotation.
For an example see https://github.com/spring-projects/spring-security/blob/7411f25474113907d4d264c38b2fe458c0d27778/web/src/test/java/org/springframework/security/web/authentication/www/BasicAuthenticationConverterTests.java#L94-L100

Reply:
I followed the convention of using AssertJ in new tests. So used `assertThatThrownBy`.